sms_toolkit/messages/profiling.py

```diff
@@ -22,16 +22,22 @@ def profile_message(message, is_for_mms=False):
     """
     encoding = utils.determine_encoding_for_string(message)
 
-    if is_for_mms:
+    if is_for_mms or encoding == constants.UCS2_ENCODING:
+        if is_for_mms:
+            single_segment_size = constants.MMS_MAX_SEGMENT_SIZE * 2
+            concat_segment_size = constants.MMS_MAX_CONCAT_SEGMENT_SIZE * 2
```

Comment:
These need to be doubled since 1600 represents 1600 2-bit characters (When encoding for UCS-2 anyway).
QUESTION: Would it be a better idea to just double them in the constant value itself (MMS_MAX_SEGMENT_SIZE = 1600 * 2)?

Reply:
If they are only used here, then the 3,200 segment size sounds more accurate, so updating the constants makes sense (especially if you include the note you provided in this comment as well).